docs/aurora/known-issues.md

```diff
@@ -90,6 +90,12 @@ There are an insufficient number of nodes are online and free for the job to sta
 
 In the event of a node going down during a job, users may encounter messages such as `ping failed on x4616c0s4b0n0: Application 047a3c9f-fb41-4595-a2ad-4a4d0ec1b6c1 not found`. The node will likely have started a reboot and won't be included in jobs again until checks pass.
 
+Use of the qsub `-V` flag(note: upper-case) is discouraged, as it can lead to startup failures.  The following message(found via pbsnodes -l):
```

Comment:
```suggestion
Use of the `qsub -V` flag (note: upper-case) is discouraged, as it can lead to startup failures.  The following message (found via `pbsnodes -l`):
```